applications/nrf5340_audio/src/bluetooth/bt_management/controller_config/Kconfig

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+rsource "Kconfig.defaults"
+
+menu "Controller config"
+
+#----------------------------------------------------------------------------#
+menu "nRF21540"
+
+config NRF_21540_ACTIVE
+	def_bool $(shields_list_contains,nrf21540_ek_fwd)
+	select EXPERIMENTAL
+	help
+	  The front end module can help boost the TX power as high as 20 dBm.
+	  Usually set through buildprog, if not set there, remember to add
+	  -DSHIELD=nrf21540_ek_fwd when building app core to include the shield
+	  overlay file.
```

Comment:
This doesn't have a prompt so it won't show up in menuconfig for the user. Do you suggest we remove this help section?

Reply:
I think so. This is a kind of information that is more relevant to nRF21540 docs rather than nRF Audio's, and it is basic configuration info the user should learn when first starting work with the NCS.